src/api/spec/models/token/workflow_spec.rb

```diff
@@ -1,165 +1,133 @@
 require 'rails_helper'
 
-RSpec.describe Token::Workflow, vcr: true do
```

Comment:
Mocking as much as possible to not test the classes which `Token::Workflow` calls. Those classes are already tested in their own spec file.

Reply:
Can you check if we have cassettes for this spec file? I see you have removed the `vcr: true` and we can remove the cassettes I think.

Reply:
I just checked, couldnt find any